lib.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] It is always great to fix the issue with the code instead of disabling eslint rule. This will ensure that your code follows best practices. This rule is very important as it will ensure you don't use a variable without defining it first.